README.md

```diff
@@ -5,3 +5,24 @@ A library for interacting with Seabug smart contracts via the Cardano Transactio
 ## Tests
 
 Use `spago test` to run the tests. Something like `nix build .#checks.<system>.seabug-contracts` can also be used, where `<system>` is something like `x86_64-linux`.
+
+## Minting
+
+The minting process currently requires some manual steps. To mint a new NFT:
+
+- Upload a new image to nft.storage (e.g. using `seabug/scripts/mint-nft.sh`)
+- Uncomment [this line](https://github.com/mlabs-haskell/seabug-contracts/blob/cda88824f87e0b961b738c66a428b7ade77454be/index.js#L39)
+- Update the image info [here](https://github.com/mlabs-haskell/seabug-contracts/blob/cda88824f87e0b961b738c66a428b7ade77454be/src/Seabug/Seabug.purs#L34)
```

Comment:
we're using branch specific links here, we'll want to change this to main when we merge up

Reply:
These are permalinks so they should continue to work (I think). The lines will probably get moved around, but this seems like a good way to do this since it should be easy to figure out what lines need to be acted on by comparing the old linked version with the updated version